keps/sig-api-machinery/1872-manifest-based-admission-webhooks/README.md

```diff
@@ -113,25 +114,7 @@ All webhooks in the manifest file need to have unique names. If a new webhook
 configuration API object with a same name as a webhook in the manifest is added,
 both webhook would be invoked. Essentially, webhooks in the manifest file will
 be treated as belonging to a different domain from the webhooks registered
-through the API.
-
-### Metrics and audit annotations
-Metrics and audit annotations for webhooks registered through the manifest file
-will be surfaced differently. This allows the administrator to unique monitor
-activity of these webhooks as they would be the only party able to take action
-against issues with these webhooks. 
-
-All admission metrics for webhooks will have an additional label `manifest_based`
-with a value `true` for webhooks registered via manifest.
-
-Audit annotations for mutation webhooks will include another field `manifestBased`
-with a value `true` for webhooks registered via manifest.
-
-```
-<<[UNRESOLVED sig-instrumentation ]>>
-get feedback on how to indicate this in metrics
-<<[/UNRESOLVED]>>
-```
+through the API. 
 
 ### New AdmissionConfig schema
 
```

Comment:
I don't see this called out, but the treatment of non-absolute file paths for `webhooksFile` should be described. Absolute paths are required for the `kubeconfigFile` field in the same config, which is the simplest approach. If relative paths are allowed, I would expect them to be treated relative to the location of the admission config file, which requires replumbing some bits of admission config loading.